prometheus-rules/prometheus-vmware-rules/values.yaml

```diff
@@ -0,0 +1,2 @@
+# The name of the Prometheus to which the alert- and aggregation rules will be assigned to.
+prometheusName: vmware
```

Comment:
What happens with the sharded prometheus setup here. Haven't checked the label yet, but i'd expect to have a label `prometheus: vmware-vc-a-0` or similar. or is this the in between approach, with the other PR upcoming to be merged after the sharded setup?

Reply:
This chart reflects the current state. We can easily extend it by looping through the same secrets file with the different targets.

Reply:
Yes.
> is this the in between approach, with the other PR upcoming to be merged after the sharded setup?